### DISCUSSION TITLE
Issue when using react-moveable along with tailwind

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [tailwindlabs/tailwindcss#9455] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.1.8

**What build tool (or framework if it abstracts the build tool) are you using?**

Vite react

**What version of Node.js are you using?**

v16.11.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**
The following repo will help in replicating the bug.
https://github.com/a-c-sreedhar-reddy/moveable-react-tailwind

```
clone the repo
npm install
npm run dev
```

**Describe your issue**

There is an issue when react-moveable is used along with tailwindcss. once `@tailwind base` ; is commented out react-movable works fine.

The following repo will help in replicating the bug.
https://github.com/a-c-sreedhar-reddy/moveable-react-tailwind

```
clone the repo
npm install
npm run dev
```

With tailwind base:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/16081083/193317704-4cb7bc8a-7d0f-4fc5-8e5e-5ac6dec89329.png">

Without tailwind base:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/16081083/193317796-29b769f0-512e-445d-9de4-a5191dad1ccf.png">

**Comment by a-c-sreedhar-reddy** [comment]
I have also raised this issue on moveable 
https://github.com/daybrush/moveable/issues/754

**Comment by thecrypticace** [comment, marked as answer]
I suspect this is a problem with the way `moveable` calculates the dimensions of elements. The problematic rule here is `img { max-width: 100% }` which causes it to break. You can work around this by adding `img { max-width: none; }` to your CSS but this seems like a fairly big change that has other implications.

Likely there's something they need to change or account for w/ regards to elements with a max-width set.
